plugin/src/main/java/org/autorefactor/refactoring/ASTHelper.java

```diff
@@ -1466,6 +1462,25 @@ public static boolean isMethod(MethodInvocation node, String typeQualifiedName,
                 methodName, parameterTypesQualifiedNames);
         return overriddenMethod != null && methodBinding.overrides(overriddenMethod);
     }
+    
+    /**
+     * Returns whether the provided method invocation invokes a method with the provided method signature.
+     * The method signature is compared against the erasure of the invoked method.
+     *
+     * @param node the method invocation to compare
+     * @param typeQualifiedName the qualified name of the type declaring the method
+     * @param methodName the method name
+     * @param parameterTypesQualifiedNames the qualified names of the parameter types
+     * @return true if the provided method invocation matches the provided method signature, false otherwise
+     */
+    public static boolean isMethod(MethodInvocation node, String typeQualifiedName,
+            String methodName, String... parameterTypesQualifiedNames) {

```

Comment:
Please move `isMethod(MethodInvocation, ...)` above `isMethod(IMethodBinding, ...)`.
This helps in reading the code top down (See Clean Code book by Uncle Bob).
And will also make the diff look better.